### PR TITLE
added thermotools recipe

### PR DESCRIPTION
**Pull request by marscher**

### CODE REVIEW

**Comment by synapticarbors** [comment]
The recipe looks good. Once the build passes on Travis, I'm happy to merge this.

**Comment by marscher** [comment]
Thank you Joshua

File: recipes/thermotools/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+    - scipy
+    - msmtools >=1.1.4
+
+test:
+  imports:
+    - thermotools 
+
+about:
+  home: http://github.com/markovmodel/thermotools
+  license: LGPL
```
Comment (jakirkham) [inline recipes/thermotools/meta.yaml]: Please add the license version in the feedstock.

File: recipes/thermotools/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,52 @@
...
+    reweighting analysis method (TRAM) and its state-discrete variant (dTRAM), Bennet acceptance ratio
+    (BAR) and its multi-state variant (MBAR), and the weighted histogram analysis method (WHAM).
+    While you can use thermotools on its own, we recommend to use it in combination with PyEMMA.
+  dev_url: https://github.com/markovmodel/thermotools
+
+extra:
+  recipe-maintainers:
+    - marscher
+    - fabian-paul 
+    - cwehmeyer
```
Comment (jakirkham) [inline recipes/thermotools/meta.yaml]: Also please let us know if you are ok being maintainers, @fabian-paul and @cwehmeyer .

Comment (marscher) [inline recipes/thermotools/meta.yaml]: These are my colleagues, so I think it is okay, if they have permissions on the feedstock ;)

File: recipes/thermotools/meta.yaml (hunk 1 above)
Comment (marscher) [inline recipes/thermotools/meta.yaml]: ok, will do in a second when the feedstock is available.

File: recipes/thermotools/meta.yaml (hunk 2 above)
Comment (cwehmeyer) [inline recipes/thermotools/meta.yaml]: I'm OK with this.

**Comment by marscher** [comment]
I the feedstock creation somehow staled?

**Comment by jakirkham** [comment]
Just waiting for the conversion job to start. It runs on Travis CI. So it sort of depends how backed up or not Travis CI is at any point in time.

File: recipes/thermotools/meta.yaml (hunk 1 above)
Comment (jakirkham) [inline recipes/thermotools/meta.yaml]: Also please [re-render]( https://github.com/conda-forge/conda-smithy#re-rendering-an-existing-feedstock ) to propagate the metadata change.

**Pull request merged**